lib/phoenix/test/conn_test.ex

```diff
@@ -494,4 +495,92 @@ defmodule Phoenix.ConnTest do
   def bypass_through(conn, router, pipelines \\ []) do
     Plug.Conn.put_private(conn, :phoenix_bypass, {router, List.wrap(pipelines)})
   end
+
+  @doc """
+  Asserts an error was wrapped and sent with the given status.
+
+  Useful for testing actions that you expect raise an error and have
+  the response wrapped in an HTTP status, with content usually renered
+  by your ErrorView.
+
+  The function accepts a status either as an integer HTTP status or
+  atom, such as `404` or `:not_found`. If an error is raised, a
+  3-tuple of the wrapped response is returned matching the
+  status, headers, and body of the response:
+
+      {404, [{"conent-type", "text/html"} | _], "Page not found"}
```

Comment:
Arghhh. You know I read this doc section an extra 3 times, just for you José. I was like I'm going to do José a solid and make this PR typo free.